packages/cpsuLambda/src/schema/format_1/transformer.ts

```diff
@@ -79,9 +82,20 @@ export function populateTemplate(
   template: string,
   prescriptionItem: itemType,
   prescriptionDetails: requestType
-): Result<BundleEntry<Task>, string> {
+): Result<BundleEntry<Task>, string> | undefined {
   const entry = JSON.parse(template) as BundleEntry<Task>
 
+  if (prescriptionItem.status === "DispensingComplete") {
+    const forbiddenStatuses = ["NotDispensed", "Expired", "Cancelled"]
```

Comment:
Does this need 'Collected' adding?

Reply:
The ticked specifies only to ignore "Expired" or "NotDispensed", but I'll check with Pete when he's back